Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM alpine:latest
+
+ENV GIT_REPOSITORY https://github.com/artefactual/atom.git
+ENV GIT_BRANCH dev/docker
+
+RUN apk update && \
+    apk upgrade && \
+    apk add -v \
+        php \
+        php-ctype \
+        php-curl \
+        php-gettext \
+        php-iconv \
+        php-json \
+        php-fpm \
+        php-memcache \
+        php-mcrypt \
+        php-mysql \
+        php-mysqli \
+        php-opcache \
+        php-pdo \
+        php-pdo_mysql \
+        php-sockets \
+        php-xml \
+        php-xmlreader \
+        php-xsl \
+        php-zip \
+        ca-certificates \
+        git \
+        openjdk8-jre-base \
+        ffmpeg \
+        imagemagick \
+        ghostscript \
+        poppler-utils \
+        nodejs \
+        make \
+        bash
+
+RUN rm -rf /var/cache/apk/*
+
+ENV FOP_VERSION 2.1
```

Comment:
2.1 has known issues that we need to resolve (unsponsored atm, #8675). Version 1.0 is the preferred version we instruct users to use for now.